libs/api/libbpf_program.cpp

```diff
@@ -193,13 +190,25 @@ bpf_program__attach_xdp(struct bpf_program* program, int ifindex)
 }
 
 struct bpf_program*
-bpf_program__next(struct bpf_program* previous, const struct bpf_object* object)
+bpf_program__next(struct bpf_program* prev, const struct bpf_object* obj)
```

Comment:
Is the parameter name change required?

Reply:
The goal is to converge code with actual libbpf code and remove code from this project over time, so trying to keep code as identical as possible to current libbpf code even if it diverges from our usual coding convention.